firestore-send-email/extension.yaml

```diff
@@ -104,11 +104,13 @@ params:
   - param: DEFAULT_FROM
     type: string
     label: Default FROM address
-    validationRegex: ^\S+@\S+\.\S+$
+    validationRegex: ^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$|^.*<(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})>$
     validationErrorMessage: Must be a valid email address
```

Comment:
Is this error message still accurate / appropriate given the new options in the regex?

Reply:
Good catch! This is checking that the the input is either: an email address, or name plus email address

Reply:
Can we produce an error message that's conditional upon which part of the regex got violated?

Reply:
ping on this thread :-)

Reply:
I don't think that this is possible. `extension.yaml` is used to setup the extension with set parameters. (e.g. `validationRegex ` & `validationErrorMessage `). However, I'll defer that call to @laurenzlong as I have no access or knowledge of `extensions` deployment context.

Reply:
Not possible unfortunately.

Reply:
directly updated the file to include "or valid name plus email address"